### ISSUE TITLE
Just a suggestion

### ISSUE THREAD

**Issue by alexalex58** [MariyaSha/GradientDescent#2]
I am following your tutorial on ML beginner. Fantastic stuff! You did a great job. Cross Entropy Loss, Gradiente Descent and so on. 
I am writing my Python code in parallel (thanks for suggesting you code). 
I realized that you are not consistent from module to module. Sometimes you warn about this and I correct immediately my code (e.g. from step activation to sigmoid) sometime you don't. I was going crazy since my values of loss were different from yours when I realized that y_target[0] was 1 in the previous module and became 0 in the gradient descent module. 

Just a suggestion in case you should come out with a renovated version of the tutorials, maintain consistency through modules. If you want I can cross check for you. 

My name is Alessandro Cagnola, Italy, alessandro.cagnola@gmail.com.
